projects/fusion-e2e/tests/components/alert/alert-component.ts

```diff
@@ -1,45 +1,59 @@
-import {Page} from '@playwright/test';
 import {AlertTestIdModifiers} from '@ironsource/fusion-ui/entities';
 import {getTestId, getTestIdSelector} from '../../global/utils';
-
-export class AlertComponent {
-    readonly page: Page;
-
-    constructor(page: Page) {
-        this.page = page;
+import {BaseComponent} from '../base-component';
+import {BaseElement} from '../../behavior';
+import {ButtonComponent} from '../button/button-component';
+
+export class AlertComponent extends BaseComponent {
+    private wrapperElement: BaseElement;
```

Comment:
Important note - it can be very problematic if the classes properties are private, since I want to be able to access the properties and execute methods that exists in baseElement or Button class directly (assuming you didn’t create a method for the same logic of course)

Reply:
This is based on the code I took from you, I assumed there was a reason you made it private, but I can remove it and it will be public by default

Reply:
Done for all project